lib/broccoli/angular-broccoli-sass.js

```diff
@@ -18,7 +18,8 @@ class SASSPlugin extends Plugin {
 
     options = options || {};
     Plugin.call(this, inputNodes, {
-      cacheInclude: [/\.scss$/, /\.sass$/]
+      cacheInclude: options.cacheInclude || [/\.scss$/, /\.sass$/],
+      cacheExclude: options.cacheExclude || undefined
```

Comment:
I could go either way. The broccoli-caching-writer that uses it has the line:
```
this._cacheExclude = options.cacheExclude || [];
```
I just figured `undefined` was "lighter" than an empty array. I'm happy to change it if you'd like.